src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.util.List;
+
+/**
+ * TaskList class that is used by duke class to deal with the list of task.
+ * It allows adding and deleting of tasks from the list.
+ */
+public class TaskList {
+    private List<Task> taskList;
```

Comment:
Would it be better to name this `tasks` instead? It may be confusing for the class name and the field name to be the same.
```suggestion
    private List<Task> tasks;
```